tests/sqllogictests/suites/base/05_ddl/05_0035_ddl_set_settings.test

```diff
@@ -0,0 +1,34 @@
+query T
+show settings like 'timestamp_output_format';
+----
+timestamp_output_format '%Y-%m-%d %H:%M:%S%.6f' '%Y-%m-%d %H:%M:%S%.6f' ["%Y-%m-%d %H:%M:%S%.3f", "%Y-%m-%d %H:%M:%S%.6f"] SESSION Specifies the display format for the TIMESTAMP data type alias. Available values include '%Y-%m-%d %H:%M:%S%.6f' and '%Y-%m-%d %H:%M:%S%.3f'. String
+
+
+statement ok
+SET timestamp_output_format='%Y-%m-%d %H:%M:%S%.3f';
+
+
+query T
+select to_timestamp('2022-02-01 00:00:00.123456789');
+----
+2022-02-01 00:00:00.123
+
+
+statement ok
+SET timestamp_output_format='%Y-%m-%d %H:%M:%S%.6f';
+
+
+query T
+select to_timestamp('2022-02-01 00:00:00.123456789');
+----
+2022-02-01 00:00:00.123456
+
+
+statement error 2803
```

Comment:
better if check for error message here
current work around `statement error 2803.*part of messges`

Reply:
Err msg in MySQL and http is diff.

Reply:
yes，the wher string after statement error  is a regrex.
so I use `.*` to handle the diff part of mysql/http handler